test/cnf_tests.jl

```diff
@@ -34,7 +34,7 @@ end
         return -mean(logpx)
     end
 
-    @testset "ADType: $(adtype)" for adtype in (Optimization.AutoForwardDiff(),
+    @testitem "ADType: $(adtype)" for adtype in (Optimization.AutoForwardDiff(),
```

Comment:
Not needed. `@testitem`s aren't meant to be stacked. Only the top level needs `@testitem`